language-server/modules/langserver-core/src/test/resources/completion/enum_decl_ctx/config/config5.json

```diff
@@ -137,7 +137,7 @@
       "detail": "Snippet",
       "sortText": "AA",
       "filterText": "public_function_main",
-      "insertText": "public function main() {\n\t${1}\n}",
+      "insertText": "function main() {\n\t${1}\n}",
```

Comment:
This case seems to be wrong. `public` should be available in the insert text here.

Reply:
Fixed with [f0d1cdc](https://github.com/ballerina-platform/ballerina-lang/pull/39678/commits/f0d1cdce329ee8533e9179772f75fb904a89f141)